litmus/OPENCL/herd/global_barrier.litmus

```diff
@@ -0,0 +1,120 @@
+OPENCL global_barrier
+
+(*
+ * https://github.com/herd/legacy/blob/master/herdtools/herd/testsuite/OpenCLTests/global_barrier.litmus
+ *)
+
+{
+  [f0] = 0;
+  [f1] = 0;
+  [tyler] = 0;
+}
+
+P0@wg 0, dev 0 (global atomic_int* f0, global atomic_int* f1) { // slave wg0, wi0
+  B1: barrier(CLK_GLOBAL_MEM_FENCE);
+
+  atomic_store_explicit(f0, 1, memory_order_release);
+
+  atomic_work_item_fence(CLK_LOCAL_MEM_FENCE, memory_order_relaxed, memory_scope_work_group);
+
+  atomic_work_item_fence(CLK_LOCAL_MEM_FENCE, memory_order_relaxed, memory_scope_work_group);
+
+  atomic_work_item_fence(CLK_LOCAL_MEM_FENCE, memory_order_relaxed, memory_scope_work_group);
+
+  atomic_work_item_fence(CLK_LOCAL_MEM_FENCE, memory_order_relaxed, memory_scope_work_group);
+
```

Comment:
Even if the original test has the fence repeated several times, this seems useless and we should clean the tests

Reply:
Removed, also for the global_barrier_mo.litmus